### PR TITLE
Plugin API implementation

### PR DESCRIPTION
**Pull request by fmmaks666**
Implements #5 

### Changes
- plugins.src - implementation for plugins
- examples - example plugins
- marilib.src - added new variables to `Var`

### TODO
- [x] Translations
- [x] Better naming
- [ ] Implementations for all Permissions
- [ ] Testing
### Future
- [x] List of plugins to import at startup
- [x] Command to manage plugins
- [ ] More APIs (Callbacks?)

### CODE REVIEW

File: src/marilib.src [hunk 1]
```diff
@@ -1,59 +1,63 @@
-//
```
Comment (h4cktoria) [inline src/marilib.src:1]: Why removing all of the // at the top of the comments? I'll change it back anyway xd It's sort of a habit commenting in MS

File: src/marilib.src [hunk 2]
```diff
@@ -1,59 +1,63 @@
...
 
-// 
+//
 // Var is something that is set at runtime and used globally
 // Don't touch unless you know what you are doing
 Var = {
     "password": null,
     "hostSession": null,
     "parameters": parseParameters(params),
+	"importPlugin": false,
```
Comment (h4cktoria) [inline src/marilib.src:38]: Not sure what is this

**Review by h4cktoria — CHANGES_REQUESTED** [review]
Overall, there are quite a few things needs to be done:

- Plugin loading logic
- Command to handle plugins
- Better variables namings
- Translations

So, perhaps, push it to prototypes for now?

**Comment by h4cktoria** [comment]
Please pull origin before pull requesting so it won't change the stuff you didn't mean to change

**Comment by fmmaks666** [comment]
These PR had issues

**Pull request closed without merging**